sdk/appconfiguration/app-configuration/samples/helloworld.ts

```diff
@@ -0,0 +1,41 @@
+import { getConnectionStringFromEnvironment, cleanupSampleValues } from "./sampleHelpers";
```

Comment:
We generally try to keep samples self-contained in a single file (minus the imports from the actual package). If we have to go across files then it starts to get overly complicated to grasp the topics.

In this case, I think you could easily reference the connection string directly from the environment variable, and leave a comment that the user should set the environment variable themselves.

I'm not sure the `cleanupSampleValues` is needed either since it looks like you're already deleting the configuration setting in the sample.

Reply:
I basically inlined the file into all of the samples. I still like cleanupSampleValues() since it makes the samples a bit more robust w/o too much clutter.